proposals/description/tip_123.ts

```diff
@@ -0,0 +1,147 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+import { ethers } from 'ethers';
+
+const tip_123: TemplatedProposalDescription = {
+  title: 'TIP_123',
+  commands: [
+    // 1. Transfer beneficiary of deprecated Rari FEI timelock to burner timelock
```

Comment:
TODO: Check ProxyAdmin ownership

Reply:
Update: Have checked all roles Core has ever granted out by querying events. Look as expected - permissions.ts config is an exhaustive representation of on-chain state

Reply:
Update: Checked all ProxyAdmin dependents. Appears that there were 5 contracts which the proxyAdmin had control over. 2 have since been transferred to Aave's control.

Remaining 3 live are:
- collateralizationOracleWrapper
- tribalChief
- vlAuraDelegatorPCVDeposit

All are fine to deprecate along with the ProxyAdmin. The 2 which have been transferred to Aave are:
- aaveTribeIncentivesController
- balancerGaugeStaker